### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by geealbers** [thegetty/rococo#58]
Alt text should be added for all images. Note any that are too complex to describe in the usual 30–40 words allowed for alt text and we'll work on coming up with an alternate solutions for them. Likely leveraging the `longdesc` attribute.
